x/staking/legacy/v040/migrate.go

```diff
@@ -129,6 +129,7 @@ func Migrate(stakingState v038staking.GenesisState) *v040staking.GenesisState {
 			MaxEntries:        uint32(stakingState.Params.MaxEntries),
 			HistoricalEntries: uint32(stakingState.Params.HistoricalEntries),
 			BondDenom:         stakingState.Params.BondDenom,
+			PowerReduction:    v040staking.DefaultParams().PowerReduction,
```

Comment:
This seems incorrect to me. This script is used for a JSON migration _to_ 0.40 (where PowerReduction was not an on-chain param). We should keep 0 here.

There should a new file ` x/staking/legacy/v043/migrate.go` to transform a 040 genesis file to a 043 genesis file

Reply:
Fixed